src/plugins/vis_types/timeseries/public/application/components/lib/index_pattern_select/switch_mode_popover.tsx

```diff
@@ -97,14 +97,14 @@ export const SwitchModePopover = ({ onModeChange, useKibanaIndices }: PopoverPro
       >
         <EuiPopoverTitle>
           {i18n.translate('visTypeTimeseries.indexPatternSelect.switchModePopover.title', {
-            defaultMessage: 'Index pattern selection mode',
+            defaultMessage: 'Data view mode',
           })}
         </EuiPopoverTitle>
         <EuiText>
           <FormattedMessage
             id="visTypeTimeseries.indexPatternSelect.switchModePopover.text"
-            defaultMessage="An index pattern identifies one or more Elasticsearch indices that you want to explore.
-            Kibana index patterns are used by default. {allowStringIndicesLabel}"
+            defaultMessage="Data view identifies one or more Elasticsearch indices that you want to explore.
+            Kibana data views are used by default. {allowStringIndicesLabel}"
```

Comment:
IMO it's apparent that data views are the default. I'd also update the data view def:

```suggestion
            defaultMessage="A data view groups and retrieves data from Elasticsearch. Disable this mode to directly query Elasticsearch indices instead.  {allowStringIndicesLabel}"
```

This is loosely based on the index patterns copy:



<img width="745" alt="Screen Shot 2021-10-01 at 2 53 09 PM" src="https://user-images.githubusercontent.com/40268737/135672458-977b5f3b-d63f-4912-808c-ec20a3790a62.PNG">